api/src/main/java/org/pmiops/workbench/config/WebMvcConfig.java

```diff
@@ -50,6 +56,19 @@ public WorkbenchEnvironment workbenchEnvironment() {
     return new WorkbenchEnvironment();
   }
 
+  @Lazy
+  @Bean
+  public GoogleCredential serviceAccountCredential() {
```

Comment:
It looks like this is something we'll use in a variety of places -- doc? This picks up the service account key that is deployed w/ the app? Or fetched on circleCI?